content/docs/composition-vs-inheritance.md

```diff
@@ -165,8 +165,8 @@ class SignUpDialog extends React.Component {
 
 ## So What About Inheritance? {#so-what-about-inheritance}
 
-At Facebook, we use React in thousands of components, and we haven't found any use cases where we would recommend creating component inheritance hierarchies.
+Facebook में हम React का हज़ार जगह प्रयोग करते है, परन्तु हमे कोई भी ऐसा केस नहीं मिला जहाँ हम आपको कौम्पोनॅन्ट्स इन्हेरिटेंस हाईरारकीस बनाने की सलाह दे।
```

Comment:
हज़ार जगह प्रयोग करते है -> does not reflect thousands of components, meaning changes

Reply:
परन्तु -> does not fit here